src/docs/release/breaking-changes/add-currentAutofillScope-to-TextInputClient.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Add currentAutofillScope to TextInputClient
+description: A new getter`TextInputClient.currentAutofillScope` was added to the `TextInputClient` interface for autofill support.
+---
+
+## Summary
+
+A new getter`TextInputClient.currentAutofillScope` was added to the
+`TextInputClient` interface.
```

Comment:
summary should include a little more detail than one sentence, maybe something like "Implementing this getting allows bla bla bla"